### DISCUSSION TITLE
开多个程序同时抢同一天的同一种票，可行么？

### DISCUSSION THREAD

**Discussion by YURE285** [mikumifa/biliTickerBuy#307] [提问专区]
没必要？还是会有被封的风险？还是什么？

**Comment by mikumifa** [comment]
没有必要

**Reply by YURE285** [reply]
OKOK

**Reply by enguangcai** [reply]
要是多开然后抢不同天的票呢？会不会因为是同一个IP且账号id是相同的被判定为抢票过于频繁，导致被封IP？

**Reply by CinnabarCHU** [reply]
每个账号在同一IP一段时间能能提交的有效请求大致是有限的，封号不至于，但是帮助不大，还是去借点账号更靠谱。

**Reply by Cloud1069** [reply]
所以这个提交次数是看ip的还是看账号的，同一个ip可以开很多账号么
